packages/webvh/package.json

```diff
@@ -26,7 +26,7 @@
     "@credo-ts/core": "workspace:*",
     "class-transformer": "^0.5.1",
     "class-validator": "0.14.1",
-    "didwebvh-ts": "2.5.5",
+    "didwebvh-ts": "2.5.6",
```

Comment:
Shouldn't we set the dependency to `^2.5.6` (see `^`), so we automatically get fixes and security updates